### ISSUE TITLE
Wall Junction Export Problem

### ISSUE THREAD

**Issue by jddaigle** [Autodesk/revit-ifc#118]
Sometime when there is a complex wall and floors junctions the conversion in IFC 2x3 give us weird result. The walls are not joined.

I used the last 19.2 converter

Here is the files

tks

[wall junction issue IFC.zip](https://github.com/Autodesk/revit-ifc/files/3340663/wall.junction.issue.IFC.zip)

[wall junction issue.zip](https://github.com/Autodesk/revit-ifc/files/3340629/wall.junction.issue.zip)

**Comment by dvrvb** [comment]
Hi Jean-Daniel,
In the model there are a few little overlaps of the walls.
If I correct the little overlap between wall 1 and wall 2, the IFC export seems to be fine (see IFC model in zip)
Regards,
Dirk
[wall junction issue.zip](https://github.com/Autodesk/revit-ifc/files/3344103/wall.junction.issue.zip)
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/41319048/60413502-708ae180-9bd5-11e9-880d-8b5ca4f5e216.png">

**Comment by jddaigle** [comment]
Thanks for your quick answer.
Yes I understand that, but how can I explain that to an architect?
The model is fine is Revit should it also be in the IFC?
Or should we always run a collision detection before the conversion?

**Comment by dvrvb** [comment]
Maybe carefully suggesting to the modelling author to take a crash course 'how to use the snap function'?
I suppose there always will be some constraints, since the conversion is a translation.
Best practise is to do an export on a regular base in order to pinpoint issues already in a very early stage. Nobody likes repairing things after the complete model is almost finished.

**Comment by jddaigle** [comment]
Yeah you are right. but I would like to make the conversion seamless for the Architects, and even they do not know what an IFC.
I'm wondering why the interaction (in that case the overlap) between the 3D objects has an impact on their shape after the conversion?

**Comment by dvrvb** [comment]
I completely understand that one should not be bothered with a lot of constraints concerning an export. But that is the ideal situation, and probably non of the existing BIM Authoring Tools can make this happen in that way.
Nevertheless, the developers probably can take a closer look into the overlap issue.

**Comment by jddaigle** [comment]
Yeah you are right. 
thanks a lot

**Comment by MuatazSafaaAlbadri** [comment]
[IFC_model_walls_.zip](https://github.com/Autodesk/revit-ifc/files/13766856/IFC_model_walls_.zip) 
Hi, I face the same issues. overlaps, not extended walls in the intersections. any guide to solve overlapping walls and extend other walls?
<img width="809" alt="Screenshot 2023-12-25 181800" src="https://github.com/Autodesk/revit-ifc/assets/152295853/630a756c-dffe-48f8-bfed-e0b69e14329e">

**Comment by ArtemHnatko** [comment]
The issue does not reproduce in the latest version of Revit 2023.

**Testing environment:**
- Autodesk Revit 2023 (v.23.1.60.0)

![Image](https://github.com/user-attachments/assets/2a263a46-d748-4012-bcde-dd606a2d88e6)
